engine/render/src/render/font.cpp

```diff
@@ -362,8 +362,8 @@ namespace dmRender
 
     static void ResetCache(HFontMap font_map, dmGraphics::HContext graphics_context, bool recreate_texture, dmRender::FontMetrics* metrics)
     {
-        font_map->m_CacheWidth = dmMath::Max(font_map->m_CacheWidth, metrics->m_MaxWidth);
-        font_map->m_CacheHeight = dmMath::Max(font_map->m_CacheHeight, metrics->m_MaxHeight);
+        font_map->m_CacheWidth = dmMath::Max(font_map->m_CacheWidth, (uint32_t)metrics->m_ImageMaxHeight);
+        font_map->m_CacheHeight = dmMath::Max(font_map->m_CacheHeight, (uint32_t)metrics->m_ImageMaxHeight);
```

Comment:
Making sure the texture is at least the size of the currently largest glyph image